src/gurt/telemetry.c

```diff
@@ -243,7 +241,7 @@ d_tm_init(int rank, uint64_t mem_size)
 	int		rc = D_TM_SUCCESS;
 
 	if ((d_tm_shmem_root != NULL) && (d_tm_root != NULL)) {
-		D_INFO("d_tm_init already completed for rank %d\n", rank);
+		D_INFO("d_tm_init already completed for srv_idx %d\n", rank);
```

Comment:
Is the terminology change correct? The variable is called rank, so we should change the variable name too, if the original understanding was wrong.

Reply:
Server index is correct.  My use of the name 'rank' is not.  I left the variable name as 'rank' for now to minimize changes for this PR.  That said, it's not a big problem to change rank -> srv_idx everywhere, so I'll do that on the next update.

Reply:
I decided to incorporate those changes now.